### PR TITLE
Refactor & Simplify AlgebroidsFromDataTables using FpCategories

### PR DESCRIPTION
**Pull request by kamalsaleh**
plus few tweaks here and there.

### CODE REVIEW

File: FpCategories/PackageInfo.g [hunk 1]
```diff
@@ -30,6 +30,20 @@ Persons := [
     Place := "Siegen",
     Institution := "University of Siegen",
   ),
+  rec(
```
Comment (mohamed-barakat) [inline FpCategories/PackageInfo.g:33]: Oh, I am so sorry I forgot that. I copied one of the existing packages in CategoricalTowers and used it as a template.

File: FpCategories/gap/Quivers.gd [hunk 2]
```diff
@@ -94,11 +94,15 @@ DeclareAttribute( "QuiverName", IsCapQuiver );
 DeclareAttribute( "NumberOfObjects", IsCapQuiver );
 
 #! @Description
-#!  Returns the labels of objects in the quiver <A>q</A>.
+#!  Returns the labels of objects all in the quiver <A>q</A>.
```
Comment (mohamed-barakat) [inline FpCategories/gap/Quivers.gd:103]: you probably mean: `all objects` rather `objects all`

File: FpCategories/gap/Quivers.gd [hunk 3]
```diff
@@ -94,11 +94,15 @@ DeclareAttribute( "QuiverName", IsCapQuiver );
...
 #! @Arguments q
 #! @Returns a list
 DeclareAttribute( "LabelsOfObjects", IsCapQuiver );
 
+#! @Description
+#!  Returns the latex-strings of all objects in the quiver <A>q</A>.
```
Comment (mohamed-barakat) [inline FpCategories/gap/Quivers.gd:109]: latex -> LaTeX

File: FpCategories/gap/Quivers.gd [hunk 4]
```diff
@@ -108,11 +112,15 @@ DeclareAttribute( "LaTeXStringsOfObjects", IsCapQuiver );
...
 
 #! @Description
-#!  Returns the labels of morphisms in the quiver <A>q</A>.
+#!  Returns the labels of all morphisms in the quiver <A>q</A>.
 #! @Arguments q
 #! @Returns a list
 DeclareAttribute( "LabelsOfMorphisms", IsCapQuiver );
 
+#! @Description
+#!  Returns the latex-strings of all morphisms in the quiver <A>q</A>.
```
Comment (mohamed-barakat) [inline FpCategories/gap/Quivers.gd:127]: latex -> LateX

**Review by mohamed-barakat — CHANGES_REQUESTED** [review]
I would like to discuss with you the last commit in detail.

File: FpCategories/gap/Quivers.gd (hunk 4 above)
Comment (kamalsaleh) [reply FpCategories/gap/Quivers.gd:127]: done!

File: FpCategories/gap/Quivers.gd (hunk 2 above)
Comment (kamalsaleh) [reply FpCategories/gap/Quivers.gd]: done!

File: Algebroids/examples/AlgebroidFromDataTables.g [hunk 5]
```diff
@@ -57,87 +49,110 @@ F := FreeCategory( q );
...
+hom_03 := BasisOfExternalHom( A.("0"), A.("3") );
 #! [ <1*a•b:(0) -≻ (3)>, <1*c•d:(0) -≻ (3)>,
 #!   <1*a•b•e:(0) -≻ (3)>, <1*c•d•e:(0) -≻ (3)>,
 #!   <1*a•b•e^2:(0) -≻ (3)>, <1*c•d•e^2:(0) -≻ (3)> ]
 alpha := 2*hom_03[1] + 3*hom_03[6];
 #! <2*a•b + 3*c•d•e^2:(0) -≻ (3)>
 LaTeXOutput( alpha );
 #! "{0}-\\left({2\\cdot {a}{b} + 3\\cdot {c}{d}{e}^{2}}\\right)\\rightarrow{3}"
-alpha = CreateMorphism( B, B.("0"), [2, 0, 0, 0, 0, 3 ], B.("3") );
+alpha = MorphismConstructor( A, A.("0"), [2, 0, 0, 0, 0, 3 ], A.("3") );
```
Comment (mohamed-barakat) [inline Algebroids/examples/AlgebroidFromDataTables.g:112]: Please enter a space in front of 2.

File: Algebroids/examples/AlgebroidFromDataTables.g [hunk 6]
```diff
@@ -57,87 +49,110 @@ F := FreeCategory( q );
...
 #! true
+CoefficientsList( alpha );
+#! [2, 0, 0, 0, 0, 3 ]
```
Comment (mohamed-barakat) [inline Algebroids/examples/AlgebroidFromDataTables.g:115]: Please add a space in front of 2.

File: Algebroids/examples/AlgebroidFromDataTables.g [hunk 7]
```diff
@@ -57,87 +49,110 @@ F := FreeCategory( q );
...
-data_tables := DataTablesOfCategory( A );;
-B := AlgebroidFromDataTables( data_tables );; # alternatively,
-B := AlgebroidFromDataTables( A );
+data_tables := DataTablesOfCategory( B );;
+data_tables[1];
+#! Q
+data_tables[2];
+#! "q(0,1,2,3)[a:0->1,b:1->3,c:0->2,d:2->3,e:3->3]"
+data_tables[3];
+#! [ [ [ [] ], [ [1] ], [ [3] ], [ [1,2],[3,4],[1,2,5],[3,4,5],[1,2,5,5],[3,4,5,5] ] ],
```
Comment (mohamed-barakat) [inline Algebroids/examples/AlgebroidFromDataTables.g:61]: Check if you do not exceed 80 characters.

File: Algebroids/examples/AlgebroidFromDataTables.g (hunk 5 above)
Comment (kamalsaleh) [reply Algebroids/examples/AlgebroidFromDataTables.g]: done!

**Review by mohamed-barakat — APPROVED** [review]

**Comment by mohamed-barakat** [comment]
Thanks.

**Pull request merged**